src/main.c

```diff
@@ -755,9 +755,9 @@ static void calculate_setpoint_target(data *d) {
     } else if (d->motor.duty_cycle > 0.05 && input_voltage > d->float_conf.tiltback_hv) {
         d->beep_reason = BEEP_HV;
         beep_alert(d, 3, false);
-        if (((d->current_time - d->tb_highvoltage_timer) > .5) ||
-            (input_voltage > d->float_conf.tiltback_hv + 1)) {
-            // 500ms have passed or voltage is another volt higher, time for some tiltback
+        if ((d->current_time - d->tb_highvoltage_timer) > 5) {
+            // It is assumed that haptic feedback is enabled!
+            // 5s have passed or voltage is another volt higher, time for some tiltback
```

Comment:
You've removed the "another volt higher" part of the condition. A case in point with these comments, self-explanatory code is better.

But more importantly, at this stage we can't make that assumption, which in my eyes just invalidates the PR.